solr/core/src/java/org/apache/solr/cloud/api/collections/CreateCollectionCmd.java

```diff
@@ -370,7 +369,7 @@ public void call(ClusterState clusterState, ZkNodeProps message, @SuppressWarnin
         log.debug("Finished create command on all shards for collection: {}", collectionName);
         // Emit a warning about production use of data driven functionality
         boolean defaultConfigSetUsed = message.getStr(COLL_CONF) == null ||
-            message.getStr(COLL_CONF).equals(DEFAULT_CONFIGSET_NAME);
+            message.getStr(COLL_CONF).equals(DEFAULT_CONFIGSET_NAME) || ConfigSetsHandler.isAutoGeneratedConfigSet(message.getStr(COLL_CONF));
```

Comment:
I don't recall why you modified this condition to include this.

Reply:
because of test; sometimes configName has `.autoCreated` suffix.

Reply:
How did the pass test prior then?  Which test?

Reply:
CollectionsAPISolrJTest#testCreateWithDefaultConfigSet fails if I remove `ConfigSetsHandler.isAutoGeneratedConfigSet(message.getStr(COLL_CONF))`